src/NLog.WindowsIdentity/NLog.WindowsIdentity.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition=" '$(TargetFrameworks)' == '' ">net46;net35;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(TargetFrameworks)' == '' ">net46;net35;netstandard2.0;netstandard2.1;net6.0-windows;net8.0-windows</TargetFrameworks>
```

Comment:
Thank you for fixing the merge conflicts. If you remove the unsupported `net6.0-windows` and fixes the [build-server-errors](https://ci.appveyor.com/project/nlog/NLog-WindowsIdentity/builds/52419567) then I will merge the pull-request.